drivers/spi/spi_context.h

```diff
@@ -291,10 +301,10 @@ void spi_context_buffers_setup(struct spi_context *ctx,
 #endif /* CONFIG_SPI_SLAVE */
 
 	LOG_DBG("current_tx %p (%zu), current_rx %p (%zu),"
-		    " tx buf/len %p/%zu, rx buf/len %p/%zu",
-		    ctx->current_tx, ctx->tx_count,
-		    ctx->current_rx, ctx->rx_count,
-		    ctx->tx_buf, ctx->tx_len, ctx->rx_buf, ctx->rx_len);
+		" tx buf/len %p/%zu, rx buf/len %p/%zu",
+		ctx->current_tx, ctx->tx_count,
+		ctx->current_rx, ctx->rx_count,
+		ctx->tx_buf, ctx->tx_len, ctx->rx_buf, ctx->rx_len);
```

Comment:
Not a big deal, but those whitespace-only changes could be part of separate commit.

Reply:
It's a really straightforward change, made just on the occasion of editing this function. I don't think it deserves a separate commit.

Reply:
It is straightforward, but I find it easier to navigate `git blame`, `git log` and (if necessary) use `git revert` / `git cherry-pick` if functional changes are separated from whitespace changes.

Reply:
yep, same here, please separate into its own commit

Reply:
Done.